### PR TITLE
Speakeasy provider RC PR

### PR DESCRIPTION
**Pull request by Alramech**

### CODE REVIEW

File: .github/workflows/generate.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    branches:
+      - '*'
+jobs:
+  generate:
+    uses: speakeasy-api/sdk-generation-action/.github/workflows/sdk-generation.yaml@v14
+    with:
+      speakeasy_version: latest
+      openapi_doc_auth_header: Authorization
+      openapi_docs: |
+        - https://raw.githubusercontent.com/Alramech/speakeasy-test/main/openapi/openapi.yaml
```
Comment (jeffchao) [inline .github/workflows/generate.yml:22]: When do we swap this with the `edge` open api yaml?

**Review by jeffchao — APPROVED** [review]
One question on the open api yaml's location. otherwise LGTM.

File: .github/workflows/generate.yml (hunk 1 above)
Comment (Alramech) [reply .github/workflows/generate.yml:22]: Ideally we'd do it when we move the openapi out of edge into it's own single file repo

Comment (jeffchao) [reply .github/workflows/generate.yml:22]: We should swap this to `edge` before we ship. Possible to do it with PAT? Is it an issue with private repositories? If so, wouldn't we have the same issue whether this spec lives inside `edge` or another repo?

Comment (Alramech) [reply .github/workflows/generate.yml:22]: I've figured out how to generate a PAT for edge but have hit a minor generation issue around connections, waitinf on resolution with the team.

**Pull request merged**